ProductService.js

```diff
@@ -0,0 +1,67 @@
+export async function getProductList(params = {}) {
+  try {
+    const url = new URL("https://panda-market-api-crud.vercel.app/products");
+    Object.keys(params).forEach((key) =>
+      url.searchParams.append(key, params[key])
+    );
+    const res = await fetch(url);
+    if (!res.ok) {
+      throw new Error(`HTTP error! status: ${res.status}`);
+    }
+    const data = await res.json();
+    return data;
+  } catch (error) {
+    console.log("에러래");
```

Comment:
에러 메시지는 개발자가 직관적으로 어떤 에러가 발생했는지 알 수 있도록 구성하는 것이 좋습니다.
예를 들면,

```suggestion
    console.log(`상품 리스트를 가져오는 도중 에러가 발생했습니다. (${error.message})`);
```